content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: KIND
+description: Instructions to setup KIND for Istio.
+weight: 21
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kind/
```

Comment:
I'm not sure aliases are needed since this is the first time this page is available and it wasn't moved from elsewhere.